utils.js

```diff
@@ -1,7 +1,7 @@
 export function calculateFontSize(text) {
   const charactersCount = text.trim().replace(/\s+/g, "").length;
-  const maxFontSize = 38;
-  const minFontSize = 15;
+  const maxFontSize = 40;
+  const minFontSize = 20;
```

Comment:
This might be a bit challenging when it comes to accessibility tools, etc. I'm not sure if we ever need to define a max font size for any good reason. 🤔

Reply:
The code actually enlarges the font size for very short flash cards but guarantees that the font size is always at least 20 pixels.  In previous sprint reviews this was well received. 
This computation is used in the CardContainer to set the `font-size`-property. Otherwise we would just set the font size to a constant value. I'm not sure, why a constant value is better for accessibility tools then, but of course I have no experience with that.